File: JavaStudy/TopCoder/Easy/A0Paper/A0Paper.java
public class A0Paper{
    public String canBuild(int[] A){
        double sum = 0;
        for (int i=0; i<A.length; i++){
        	sum += (1/Math.pow(2,i))*A[i];
            if (sum >= 1) return "Possible";
        }
        return "Impossible";
    }
}

File: C#Study/TopCoder/Easy/A0Paper/A0Paper.cs
using System;

public class A0Paper
{
    public string CanBuild(int[] A)
    {
        double sum = 0;
        for (int i = 0; i < A.Length; i++)
        {
            sum += (1 / Math.Pow(2, i)) * A[i];
            if (sum >= 1)
                return "Possible";
        }
        return "Impossible";
    }
}
